config.yaml

```diff
@@ -1,3 +1,14 @@
+##
+## GPU and GSTREAM using Jetson Nano
+#
+
+# Use Jetosn Nano GPU
+USE_GPU:                                True
```

Comment:
This config flag seems no to be used anywhere, did you forget a change or do you plan to add something more?

Reply:
I use Jetpack 4.6.2, tensorflow 2.6.2 and CUDA 10.2. It works very vell, but I remember that it took some effort to get it up and running. So GPU is working for me.

Reply:
That fantastic! It didn't work for me and i downgrade jetpack.